ros2pkg/ros2pkg/resource/ament_cmake/CMakeLists.txt.em

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.5)
+project(@(project_name))
+
+# Default to C99
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 99)
+endif()
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+@[if dependencies]@
+@[for deb in dependencies]@
+find_package(@deb REQUIRED)
+@[end for]@
+
+@[if dependencies or create_cpp_library]@
+include_directories(
+@[if create_cpp_library]@
+  include
+@[end if]@
+)
+
+@[else]@
+# uncomment the following section in order to fill in
+# further dependencies manually.
+# find_package(<dependency> [<REQUIRED>|<QUIET>])
+
+@[end if]@
+@[end if]@
+@[if create_cpp_library]@
+add_library(${PROJECT_NAME} src/@(cpp_library_name))
```

Comment:
I would suggest to wrap before the source file to encourage users to list them on separate lines when the list grows.